util/pastPerformance/testReleasesPerformance

```diff
@@ -47,6 +47,14 @@ fi
 export CHPL_HOME_ORIG=$CHPL_HOME
 export CHPL_TEST_UTIL_DIR=$CHPL_HOME_ORIG/util
 
+if [[ -z $CHPL_TEST_VENV_DIR && ! -z $CHPL_HOST_PLATFORM ]]; then
+    export CHPL_TEST_VENV_DIR=$CHPL_HOME/third-party/chpl-venv/install/$CHPL_HOST_PLATFORM/chpl-virtualenv
+fi
+if [[ ! -d $CHPL_TEST_VENV_DIR ]]; then
+    echo "ERROR: incorrect or missing CHPL_TEST_VENV_DIR or CHPL_HOST_PLATFORM"
```

Comment:
I wasn't suggesting doing full error handling (this is designed for developers, not users), merely echoing $CHPL_TEST_VENV_DIR as part of the error message.  I know that if I hit this error, the first thing I'd do is edit the script to echo that variable so that I could start to determine where things went wrong and figuring out which of those three cases I'd hit.  Why force each developer to make that mod if/when they run into this error?